### ISSUE TITLE
number of `karaktere, fjalë, shenjime`

### ISSUE THREAD

**Issue by AndiBraimllari** [OpenCovenant/quill#119]
The generated numbers of `karaktere, fjalë, shenjime` are different than those of many other applications. Check the RegEx and discuss on what the generated numbers should be.

**Comment by AndersonCeci** [comment]
I have noticed that when I write on the editor it grabs an extra word, I have tested in on other apps too like this app "https://wordcounter.net/character-count" 
This is the text I have been testing them:
Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
On penda editor it grabs an extra word "fjale"  like 92 words
On the other app its 91 words thats the only difference
